addons/website/views/snippets/s_showcase.xml

```diff
@@ -5,52 +5,41 @@
+                            <i class="s_showcase_icon fa fa-star-o me-auto rounded float-start" role="img"/>
+                            <div class="px-2 overflow-hidden">
+                                <h5 class="s_showcase_title">Highlights Key Attributes</h5>
+                                <p>A feature section allows you to clearly showcase the main benefits and unique aspects of your product.</p>
                             </div>
-                            <p>A short description of this great feature.</p>
                         </div>
-                    </div>
-                </div>
-                <div class="col-1 o_not_editable">
-                    <div class="w-50 h-100 border-end o_not_editable"/>
-                </div>
-                <div class="col-lg">
-                    <div class="row">
-                        <div class="col-lg-12 pt24 pb24" data-name="Block">
-                            <div class="s_showcase_title d-flex align-items-center mb-2">
-                                <i class="s_showcase_icon fa fa-2x fa-heart text-secondary me-3"/>
-                                <h3>Another feature</h3>
+                        <div class="col-12 pt8 pb8" data-name="Feature">
+                            <i class="s_showcase_icon fa fa-user-o me-auto rounded float-start" role="img"/>
+                            <div class="px-2 overflow-hidden">
+                                <h5 class="s_showcase_title">Engages Visitors</h5>
+                                <p>It captures your visitors' attention and helps them quickly understand the value of your product.</p>
                             </div>
-                            <p>A short description of this great feature.</p>
                         </div>
-                        <div class="col-lg-12 pt24 pb24" data-name="Block">
-                            <div class="s_showcase_title d-flex align-items-center mb-2">
-                                <i class="s_showcase_icon fa fa-2x fa-gift text-secondary me-3"/>
-                                <h3>Last Feature</h3>
+                        <div class="col-12 pt8 pb8" data-name="Feature">
+                            <i class="s_showcase_icon fa fa-heart-o me-auto rounded float-start" role="img"/>
+                            <div class="px-2 overflow-hidden">
+                                <h5 class="s_showcase_title">Boosts Conversions</h5>
+                                <p>Organizing and presenting key information effectively increases the likelihood of turning your visitors into customers.</p>
                             </div>
-                            <p>A short description of this great feature.</p>
                         </div>
                     </div>
                 </div>
+                <div class="col-lg-6">
+                    <img src="/web/image/website.s_showcase_default_image" class="img img-fluid mx-auto rounded" alt=""/>
```

Comment:
I'd suggest to move the image to the right by default, wdyt?

| this PR | proposal |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/c33acf6e-2652-4dfd-8431-5eacf71c9370) | ![image](https://github.com/user-attachments/assets/fcd6c01c-34f8-461a-bb7a-a0844efdc898) | 


```suggestion
                    <img src="/web/image/website.s_showcase_default_image" class="img img-fluid ms-auto float-end rounded" alt=""/>
```